content/altinn-studio/guides/monitor-and-instrument/_index.nb.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Monitorer og instumenter
+linktitle: Monitorer og instumenter
+description: Kom i gang med verktøyene for instrumentering og monitorering i Altinn
+toc: true
+weight: 990
+---
+
+{{% notice info %}}
+Denne siden refererer til konfigurasjon når [OpenTelemetry (OTel)](https://opentelemetry.io/) er i bruk i appen fra v8 og nyere.
+Det eksisterende Application Insights SDK oppsettet er utgått, og vil fjernes i neste hovedversjon av Altinn.App biblioteker
+{{% /notice %}}
+
+## Konfigurering av en app
+
+Det nye monitorerings- og instrumenteringsoppsettet basert på OpenTelemetry kan aktiveres ved å sette 
+`UseOpenTelemetry` til `true` i `appsettings.json` eller ekvivalent.
+
+{{< highlight json "linenos=false,hl_lines=3" >}}
+{
+  "AppSettings": {
+    "UseOpenTelemetry": true
+  }
+}
+{{< / highlight >}}
+
+Når appen kjøres med denne instillingen så vil Altinn.App biblioteket sende telemetri til localtest ved lokal kjøring
+og til Azure Monitor ved kjøring i et miljø.
+
+[Les mer om konfigurasjonsmuligheter på referansesiden for konfigurasjon](/nb/altinn-studio/reference/monitoring/configuration).
+
+## Egendefinert instrumentering
```

Comment:
Kanskje legge til noe om hva eksempelet innebærer? Hvilken funksonalitet får du hvis du følger eksempelet og lar koden bli igjen i appen din? En teller for hvor mange ganger appen din starter opp ?

Reply:
Prøvde å legge til mer info her. Hva synes du? Det er kanskje litt kjipt eksempel (ikke så veldig "ekte"). Jeg tenkte til å begynne med at det var greit med noe man bare kunne putte i `Program.cs`? Ellers kunne vi jo laget et eksempel hvor man implementerer et ordentlig interface også (en forenklet versjon av det på instrumenterings-siden er jo en mulighet)